src/ai/backend/manager/api/service.py

```diff
@@ -303,12 +303,12 @@ class ServiceConfigModel(LegacyBaseRequestModel):
             "Specifications about extra VFolders mounted to model service session. "
             "MODEL type VFolders are not allowed to be attached to model service session with this option."
         ),
-        default={},
+        default_factory=dict,
     )
 
-    environ: Optional[dict[str, str]] = Field(
+    environ: dict[str, str] = Field(
         description="Environment variables to be set inside the inference session",
-        default=None,
+        default_factory=dict,
```

Comment:
Shouldn't we receive it as none from the API and convert it when passing it to the internal service logic? I don't think we necessarily need to change the API layer.